config/routes.rb

```diff
@@ -6,6 +6,7 @@
       resources :salons
       resources :users
       resources :visits
+      resources :categories
```

Comment:
You should write so only if you have all default actions in your controller, so, if you will delete `edit` action you should write `only` param